implementations/python/mzlib/annotation.py

```diff
@@ -149,6 +152,17 @@ class IonAnnotationBase(object, metaclass=SeriesLabelSubclassRegisteringMeta):
     series_label = None
     _molecule_description_fields = {}
 
+    series: str
+    neutral_losses: list
```

Comment:
`List`

Same for `adducts` two lines down and return values of `adduct()` and `neutral_loss()`.